sdk/contentsafety/Azure.AI.ContentSafety/src/Customization/TextBlocklist.Serialization.cs

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable disable
+
+using System.Text.Json;
+using Azure;
+using Azure.Core;
+
+namespace Azure.AI.ContentSafety
+{
+    public partial class TextBlocklist : IUtf8JsonSerializable
+    {
+        void IUtf8JsonSerializable.Write(Utf8JsonWriter writer)
+        {
+            writer.WriteStartObject();
+            writer.WritePropertyName("blocklistName"u8);
```

Comment:
We should not write this value if it hasn't changed.

Also - doesn't it go in the URI and not the request body?  Why would we want it to appear in both places?

In principle, PATCH payloads shouldn't contain anything if nothing has changed on the model.  Of course, in that case, they also shouln't be sent since they don't do anything, but I don't think we discussed with architects whether we wanted to do any validation around that - I would expect we would not, since we try not to do client-side validation.